### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by stunnerboyone**
[DEMO LINK](https://stunnerboyone.github.io/react_tabs-js/)

### CODE REVIEW

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]

File: src/components/Tabs/Tabs.jsx [hunk 1]
```diff
@@ -1 +1,41 @@
...
+              <li
+                className={cn({ 'is-active': selectedTab.id === id })}
+                data-cy="Tab"
+                key={id}
+              >
+                <a
+                  href={`#${id}`}
+                  data-cy="TabLink"
+                  onClick={() => id !== selectedTabId
+                    && onTabSelected(tab)}
```
[the remark this answers is not in the document]
Comment (stunnerboyone) [reply src/components/Tabs/Tabs.jsx:23]: How I can set tab as param if it was destructured?

[the remark this answers is not in the document]
Comment (stunnerboyone) [reply src/components/Tabs/Tabs.jsx]: How can I add tab as a parameter if it was destructured?

File: src/components/Tabs/Tabs.jsx [hunk 2]
```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
```
[the remark this answers is not in the document]
Comment (VitaliyBodnar) [reply src/components/Tabs/Tabs.jsx:3]: agree, please rename it

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

File: src/components/Tabs/Tabs.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (stunnerboyone) [reply src/components/Tabs/Tabs.jsx:3]: image
If I rename tests doesn't pass

[the remark this answers is not in the document]
Comment (stunnerboyone) [reply src/components/Tabs/Tabs.jsx:3]: Pass the `onTabSelected` callback to the `Tabs`, it should be called whenever the user selects another tab.
   (Don't call the callback if the tab was not changed)

**Review by Esceype — APPROVED** [review]
GJ🚀🚀🚀

**Pull request closed without merging**